backends/backend_cryptomb_common.h

```diff
@@ -379,7 +379,7 @@ __attribute__((aligned(64))) static const int8u pSeed[]  = "\x00\x00\x00\x00\x00
                                                            "\x00\x00";
 
 /* XOR block */
-__INLINE void XorBlock(const void* pSrc1, const void* pSrc2, void* pDst, int len)
+static void XorBlock(const void* pSrc1, const void* pSrc2, void* pDst, int len)
```

Comment:
Why did we delete inline? Should it also be deleted here? https://github.com/ElenaTyuleneva/acvpparser/blob/07b6805cab240f1161a7617f65d6db0e15aa2276/backends/backend_ippcrypto_common.h#L28

Reply:
`__INLINE` keyword wasn't recognized by the compiler, since this keyword is compiler-specific. It was a fix for compilation, which also prevented double-declaration error.  Thanks for finding another place, I'll change inline to static there as well(we don't have application size concerns, since it's just the testing binary).

Reply:
Done as a part of [ed4a304](https://github.com/ElenaTyuleneva/acvpparser/pull/1/commits/ed4a304aa3ee43acf57db8a092e3602c7c00a392)